### PR TITLE
Replace default wallpaper for global English

### PR DESCRIPTION
**Pull request by erikos**
Add two new desktop backgrounds and use one as the default one.

https://phabricator.endlessm.com/T28188

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -46,7 +46,7 @@ uninstall_pattern = \
 
 install-data-local:
 	@$(call install_pattern,$(images_instdir),$(images_srcdir)/*)
-	cp $(DESTDIR)$(images_instdir)/bg_global_02.jpg $(DESTDIR)$(pkgdatadir)/desktop-background-C.jpg
+	ln -fs $(images_instdir)/bg_global_10.jpg $(DESTDIR)$(pkgdatadir)/desktop-background-C.jpg
```
Comment (wjt) [inline Makefile.am]: I checked the Git history and there actually is a good reason why this is a copy, not a symlink, though it was buried in a code review comment.

This file ends up in a different package, `eos-default-background`, and the `eosinstaller` ostree only includes `eos-default-background`, not `eos-media`, which keeps the image size down. See db54b82c5aa2d398f19d6f217e32a1d3b3470ade, https://github.com/endlessm/eos-theme/pull/234#discussion_r63256233 and https://phabricator.endlessm.com/T10782

Please change this to something like:

```suggestion
	# This file is shipped in a separate package, eos-default-background, so cannot be a symlink
	cp $(DESTDIR)$(images_instdir)/bg_global_10.jpg $(DESTDIR)$(pkgdatadir)/desktop-background-C.jpg
```

Comment (erikos) [reply Makefile.am]: Thanks, for digging. Changed this.

File: Makefile.am [hunk 2]
```diff
@@ -46,7 +46,8 @@ uninstall_pattern = \
 
 install-data-local:
 	@$(call install_pattern,$(images_instdir),$(images_srcdir)/*)
-	cp $(DESTDIR)$(images_instdir)/bg_global_02.jpg $(DESTDIR)$(pkgdatadir)/desktop-background-C.jpg
+        # This file is shipped in a separate package, eos-default-background, so cannot be a symlink
```
[the remark this answers is not in the document]
Comment (erikos) [reply Makefile.am]: Yeah, seen as well while looking at the diff in the web interface :P

[the remark this answers is not in the document]
Comment (wjt) [reply Makefile.am]:
```
Makefile.am:49: error: '#' comment at start of rule is unportable
```

whaaaaatttt?

**Pull request merged**